docs/consume-packages/consuming-packages-authenticated-feeds.md

```diff
@@ -0,0 +1,98 @@
+title: Consuming packages from authenticated feeds
+description: Consuming packages from authenticated feeds in all NuGet client scenarios
+author: nkolev92
+ms.author: nikolev
+ms.date: 02/28/2020
+ms.topic: conceptual
+---
+
+# Consuming packages from authenticated feeds
+
+In addition to the nuget.org [public feeds](https://api.nuget.org/v3/index.json), the NuGet clients have the ability to interact with file feeds and private http feeds.
+
+To authenticate with private feeds, the 2 approaches are:
+
+* Add credentials in the [NuGet.Config](../reference/nuget-config-file.md#packagesourcecredentials)
+* Authenticate using one of the many extensibility models depending on the client used.
+
+## NuGet clients' authentication extensibility
+
+In the NuGet clients' the authentication responsibility is pushed to the private feed provider.
+Specifically all NuGet clients have extensibility methods, that's either a Visual Studio extension or a plugin that can communicates with NuGet to retrieve credentials.
+
+### Visual Studio
+
+In Visual Studio, NuGet exposes an interface that feed providers can implement and provide to their customers. For more information on [how to create](../reference/extensibility/NuGet-Credential-Providers-for-Visual-Studio.md) a Visual Studio credential provider.
+
+#### Available NuGet credential providers for Visual Studio
+
+There is a credential provider built into the Visual Studio NuGet extension to support Visual Studio Team Services.
+
+Available plug-in credential providers include:
+
+* [MyGet Credential Provider for Visual Studio](http://docs.myget.org/docs/reference/credential-provider-for-visual-studio)
+
+### nuget.exe
+
+When `nuget.exe` needs credentials to authenticate with a feed, it looks for them in the following manner:
+
+1. Look for credentials in `Nuget.Config` files.
+1. Use V2 plug-in credential providers subject to the ordering below.
+1. Use V1 plug-in credential providers, subject to the ordering below.
+1. NuGet then prompts the user for credentials on the command line.
+
+#### nuget.exe and V2 credential providers
+
+In version `4.8` NuGet defined an authentication plugin mechanism that works in all clients.
+For the installation and discovery of those providers, refer to [NuGet cross platform plugins](../reference/extensibility/NuGet-Cross-Platform-Plugins.md#plugin-installation-and-discovery).
+
+#### nuget.exe and V1 credential providers
+
+In version `3.3` NuGet introduced the first version of authentication plugins.
+For the installation and discovery of those providers refer to [nuget.exe credential providers](../reference/extensibility/nuget-exe-Credential-Providers.md#nugetexe-credential-provider-discovery)
+
+#### Available credential providers for nuget.exe
+
+1. [Azure DevOps V2 Credential Providers](azure/devops/artifacts/nuget/nuget-exe?view=azure-devops#add-a-feed-to-nuget-482-or-later) or [Azure Artifacts Credential Provider](https://github.com/microsoft/artifacts-credprovider)

```

Comment:
should this be two items?

Reply:
It's one, they are the same thing, but one is if you are using the Azure DevOps tasks, but the other one is if you want to get the plugin your self. 

Thoughts/ideas on how to improve or you think this is fine?

Reply:
That's fine, then :)